massa-models/src/amount.rs

```diff
@@ -9,32 +9,47 @@ use std::str::FromStr;
 
 pub const AMOUNT_ZERO: Amount = Amount::from_raw(0);
```

Comment:
Why? How? Where? Is that constant used?

Reply:
we could replace it with a zero() function constructor if you wish (like here https://docs.rs/num/latest/num/traits/trait.Zero.html#tymethod.zero )... but the result is kinda the same

Reply:
Note that it could also be defined as a Default. But when you want a zero somewhere, I find using AMOUNT_ZERO much more explicit and readable than "Default::default()".

Reply:
Maybe a const function Amount::zero() would actually be nicer than a random constant in the config

Reply:
good idea, with `const fn` !

Reply:
I close that and resolve it in a minute forward